StortSpelProjekt/Game/src/Gamefiles/Components/UpgradeComponents/UpgradeRangeTest.cpp

```diff
@@ -0,0 +1,26 @@
+#include "UpgradeRangeTest.h"
+#include "stdafx.h"
+#include "../ECS/Entity.h"
+
+component::UpgradeRangeTest::UpgradeRangeTest(Entity* parent)
+	:UpgradeComponent(parent)
+{
+	// name the upgrade!
+	SetName("UpgradeRangeTest");
+	// set the upgrade type
+	SetType(E_Type::RANGE);
+	// increase upgrade level
+	IncreaseLevel();
+}
+
+component::UpgradeRangeTest::~UpgradeRangeTest()
+{
+}
+
+void component::UpgradeRangeTest::OnHit()
+{
+	Log::Print("UpgradeRangeTest OnHit called\n");
```

Comment:
If this print is for testing, then its all good. But we should try to avoid to print stuff during runtime in release mode if its not really necessary.